spec/menu_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'menu'
+
+describe "MENU" do
+  
+  it 'is a hash' do
+    expect(MENU).to be_a Hash
+   end
```

Comment:
Lint/BlockAlignment: `end` at 7, 3 is not aligned with `it 'is a hash' do` at 5, 2.